editor/SettingsWindow.py

```
@@ -447,6 +448,33 @@ def _create_keyboard_tab(self):
         layout.addLayout(rebind_grid)
         layout.addStretch()
 
+    def _create_kiosk_tab(self):
+        """Package Player / Kiosk Mode settings."""
+        widget = QWidget()
+        layout = QVBoxLayout(widget)
+        self.tabs.addTab(widget, "Package Player")
+
+        mode_group = QGroupBox("Window Mode")
+        mode_layout = QFormLayout()
+        self.kiosk_mode_combo = QComboBox()
```

Ruff (0.15.12)

[error] Undefined name `QComboBox`

(F821)